QUICK_FIX_NGROK.md

```
@@ -0,0 +1,99 @@
+
+1. **Start backend first** (in one terminal):
+   ```bash
+   # Start dev services
+   docker compose -f ./.docker/dev/compose.yml up -d
+   
+   # Start backend (runs on port 3010)
+   yarn affine server dev
+   ```
+
+2. **Wait for backend to start** - you should see:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)